[2025-02-28T23:40:11.016+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_db_dag.process_daily_data scheduled__2025-02-28T23:38:00+00:00 [queued]>
[2025-02-28T23:40:11.021+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_db_dag.process_daily_data scheduled__2025-02-28T23:38:00+00:00 [queued]>
[2025-02-28T23:40:11.021+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-28T23:40:11.029+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): process_daily_data> on 2025-02-28 23:38:00+00:00
[2025-02-28T23:40:11.035+0000] {standard_task_runner.py:57} INFO - Started process 3784 to run task
[2025-02-28T23:40:11.040+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_to_db_dag', 'process_daily_data', 'scheduled__2025-02-28T23:38:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpapqog20v']
[2025-02-28T23:40:11.043+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask process_daily_data
[2025-02-28T23:40:11.084+0000] {task_command.py:410} INFO - Running <TaskInstance: api_to_db_dag.process_daily_data scheduled__2025-02-28T23:38:00+00:00 [running]> on host d71706374825
[2025-02-28T23:40:11.148+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_db_dag' AIRFLOW_CTX_TASK_ID='process_daily_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T23:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T23:38:00+00:00'
[2025-02-28T23:40:13.981+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/pyspark/context.py:317 FutureWarning: Python 3.7 support is deprecated in Spark 3.4.
[2025-02-28T23:40:20.954+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-28T23:40:20.991+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=api_to_db_dag, task_id=process_daily_data, execution_date=20250228T233800, start_date=20250228T234011, end_date=20250228T234020
[2025-02-28T23:40:21.070+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-28T23:40:21.132+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
